contracts/core/abstract/RequestHandlerBase.sol

```diff
@@ -7,6 +7,12 @@ import {IRequestHandler} from "../../interfaces/IRequestHandler.sol";
 /// @title RequestHandler
 /// @notice Implements the RequestHandlerBase for managing Seda protocol requests
 abstract contract RequestHandlerBase is IRequestHandler {
+    // ============ Constants ============
+    uint256 internal constant MIN_REPLICATION_FACTOR = 1;
+    uint256 internal constant MIN_GAS_PRICE = 1_000;
+    uint256 internal constant MIN_EXEC_GAS_LIMIT = 10_000_000_000_000;
+    uint256 internal constant MIN_TALLY_GAS_LIMIT = 10_000_000_000_000;
```

Comment:
I think these can be more readable by having a constant `TERA_GAS = 1_000_000_000_000` and then these would be equal to `10 * TERA_GAS`.